docs/src/SUMMARY.md

```diff
@@ -81,6 +81,7 @@
         * [Memory](spec/data_layout/memory/index.md)
             * [Sequence types in memory](spec/data_layout/memory/sequence_types_in_memory.md)
     * [Function calls](spec/function_calls.md)
+    * [Precompiles](spec/precompiles.md)
```

Comment:
Since precompiles are part of the std library, I believe this should sit outside of `spec` as spec is supposed to as close as possible a language specification (inspired by https://doc.rust-lang.org/stable/reference/introduction.html) and hence not cover std library stuff. I think that this should either become a sub of "Using Fe" or become a sub of a completely new item but I'm not sure what that would be (maybe "Standard Library"). The more I think about it, I think I like the idea of organizing this under a "Standard Library" section.